src/policy/fees_util.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license. See the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+#include <policy/fees_util.h>
+#include <primitives/transaction.h>
+
+TxAncestorsAndDescendants GetTxAncestorsAndDescendants(const std::vector<RemovedMempoolTransactionInfo>& transactions)
+{
+    TxAncestorsAndDescendants visited_txs;
+    for (const auto& tx_info : transactions) {
+        const auto& txid = tx_info.info.m_tx->GetHash();
+        std::set<Txid> tx_set{txid};
+        visited_txs.emplace(txid, std::make_tuple(tx_set, tx_set));
```

Comment:
Interesting that `tx_set` is passed by value here and 2 separate copies are made at the time of calling, which is the need here for separate ancestors and descendants copies.

Reply:
yes, this is gone now.